src/Symfony/Component/DependencyInjection/Tests/ContainerBuilderTest.php

```diff
@@ -50,6 +51,8 @@
 
 class ContainerBuilderTest extends TestCase
 {
+    use ExpectDeprecationTrait;
```

Comment:
TODO after https://github.com/symfony/symfony/pull/36468 is merged

Reply:
it is now